testapp/src/main/java/com/rakuten/tech/mobile/testapp/rat_wrapper/CustomButtonViewWithArrow.kt

```diff
@@ -0,0 +1,65 @@
+package com.rakuten.tech.mobile.testapp.rat_wrapper
```

Comment:
I think, we can move this package under `com.rakuten.tech.mobile.testapp.analytics` ?

Reply:
Changed the directory 🙂